lib/foods.js

```diff
@@ -0,0 +1,17 @@
+$(document).ready(function(){
+  $.ajax({
+    type: "get",
+    url: "http://localhost:3000/api/v1/foods"
+  }).done(function(foods){
+    var foodsArray = foods;
+    foodsArray.forEach(function(food){
+      $(".food-table").append("<tr><td name='food-cell'>"
+      + food.name
+      + "</td><td name='calorie-cell'>"
+      + food.calories
+      + "</td><td name='delete-cell'><a href='webpack-dev-server/index.html'><img src='/lib/images/delete_red.png' id='delete-food-img'/></a></td></tr>");
```

Comment:
Can we refactor this call to be more like the one we saw in the AJAX refresher in class today? You can just check out the version [here](https://github.com/katiekeel/quantified-self/blob/story_2_add_foods/lib/foods.js) in my branch.

Reply:
Absolutely! Working on the ajax refactor + javascript PORO refactors now.